app/src/js/reducers/api.js

```diff
@@ -27,40 +26,38 @@ export const initialState = {
 };
 
 export default createReducer(initialState, {
-  [DELETE_TOKEN]: (state) => {
-    set(state, 'tokens.token', null);
+  [DELETE_TOKEN]: ({ tokens }) => {
```

Comment:
I don't love destructuring the input arguments, but I see you did it everywhere.  Can you tell me why this is clearer or better than using draftState as the input argument?

Reply:
Would it be so bad as:
 ```
 [DELETE_TOKEN]: ( draftState ) => {
    setToken('');
    draftState.tokens.token = null;
  },
```

Reply:
Yeah, I'm not convinced I like it either. There was one place where it seemed easier/nicer at the time, so then I used it throughout that file (don't recall which one), and then realized I should do it everywhere for consistency.

However, I think it is perhaps not so good, so I'll get rid of all of the argument destructuring. In fact, at one point I do recall forgetting what the original arguments were supposed to be because everything was destructured, so that also seems to be a strong argument for getting rid of it.

Thanks for calling it out!

Reply:
Oh, I'll also get rid of that inline assignment too.